src/components/SectionTable/SectionTableButtons.js

```diff
@@ -58,6 +69,12 @@ export const ScheduleAddCell = withStyles(styles)((props) => {
         }
 
         if (scheduleIndex !== -1) {
+            if (specificSchedule) {
+                logAnalytics({
+                    category: analyticsEnum.classSearch.title,
+                    action: analyticsEnum.classSearch.actions.ADD_SPECIFIC,
+                });
+            }
```

Comment:
What is the point of the specificSchedule check?

Reply:
It's to see how often people are using the dropdown to add to a specific schedule, versus just adding to the current schedule with the plus
![image](https://user-images.githubusercontent.com/48658337/168930261-c17014f8-62c9-4b71-9a7b-be1cef2e2cff.png)
blue - specific schedule.
red - current schedule

Reply:
Ah nice, this is a good analytic to capture! I'd love to see if people are actually using it.